lib/pat.c

```diff
@@ -890,6 +950,14 @@ grn_pat_wal_add_entry(grn_ctx *ctx, grn_pat_wal_add_entry_data *data)
     usage = "deleting entry";
     rc = grn_pat_wal_add_entry_delete_entry(ctx, data, &used);
     break;
+  case GRN_WAL_EVENT_DEFRAG_KEY:
+    usage = "defrag key";
+    rc = grn_pat_wal_add_entry_defrag_key(ctx, data, &used);
+    break;
+  case GRN_WAL_EVENT_DEFRAG_CURRENT_KEY:
+    usage = "defrag curr_key";
```

Comment:
ここはhuman readableなやつでいいっす。

```suggestion
    usage = "defrag current key";
```